maven.gradle

```diff
@@ -1,12 +1,19 @@
-import com.infinum.maven.SonatypeConfiguration
-
-SonatypeConfiguration.metaClass.constructor = { ->
```

Comment:
Setup with this didn't work for in here. If I'd execute task from command line it would look for the props file in the root directory, but task would be executed by double clicking from AS then it would look for the file in project directory.
As this was loaded in kts file that wasn't a task or anything it wasn't possible to determine root directory. Because of that I removed that code and load everything in here.